modules/activity_tracker/variables.tf

```diff
@@ -105,13 +105,14 @@ variable "cos_bucket_endpoint" {
 # COS Targets
 variable "cos_targets" {
   type = list(object({
-    endpoint                   = string
-    bucket_name                = string
-    instance_id                = string
-    api_key                    = optional(string)
-    service_to_service_enabled = optional(bool, false)
-    target_region              = optional(string)
-    target_name                = string
+    endpoint                          = string
+    bucket_name                       = string
+    instance_id                       = string
+    api_key                           = optional(string)
+    service_to_service_enabled        = optional(bool, false)
```

Comment:
I think this should default to `true` actually, since this is the recommended approach